scripts/cluster/setup_master_node.go

```diff
@@ -263,7 +290,14 @@ func ConfigureMagicDNS() error {
 // Deploy Istio pods
 func DeployIstioPods() error {
 	utils.WaitPrintf("Deploying istio pods")
-	_, err := utils.ExecShellCmd("kubectl apply -f https://github.com/knative/net-istio/releases/download/knative-v%s/net-istio.yaml", configs.Knative.KnativeVersion)
+	
+	loaderIstioControllerPath, err := utils.GetVHiveFilePath(path.Join("configs/knative_yamls", "loader_istio_controller.yaml"))
```

Comment:
This file is not found in the main vhive branch.

I propose search for this file in the repo, in case it is not found, download and use the default one. Same goes for the serving-core stuff. In stock-only case, running these scripts __not__ from the loader should install default yamls.